### ISSUE TITLE
sparse_loss

### ISSUE THREAD

**Issue by zhangxinpe** [Lluvia-Tang/MGFN-main#3]
loss = self.opt.gamma * Lc + self.opt.theta * Lr + self.opt.lex * Ls + self.opt.sparseL * sparse_loss

                loss.backward()

May I ask if there is no loss in your code, is it a code error? It's not the same as your paper
